tests/tasks/test_example.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+from openedx_webhooks.tasks.example import add
+
+
+def test_add():
+    assert add(5, 6) == 11
+
+
+if __name__ == '__main__':
```

Comment:
We don't need this in every test file, do we? I'm not a heavy pytest user, but I would have thought it could take care of this for us.